### PR TITLE
Combined About and Landing Page

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.10/enterprise/_index.md [hunk 1]
```diff
@@ -4,17 +4,35 @@ menu:
...
+These Nodes run Corda applications [(CorDapps)](../../enterprise/cordapps/cordapp-overview.html), and transact between Nodes using public or
+confidential identities.
+
+When one or more Nodes are involved in a transaction, the transaction must be notarised. [Notaries](../../enterprise/notary/ha-notary-service-overview.md) are a specialised type
+of Node that provides uniqueness consensus by attesting that, for a given transaction, it has not already signed other
+transactions that consumes any of the proposed transaction’s input states.
+
+To learn more about Corda, select any of the child pages of this **About Corda** section.
+
+## Corda Enterprise
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/_index.md:30]: Doesn't seem like a great sub-heading? Similar to the H1 so not clear what the section contains?

File: content/en/platform/corda/4.10/enterprise/_index.md [hunk 2]
```diff
@@ -4,17 +4,35 @@ menu:
...
+    name: Corda Enterprise 4.10
 project: corda
 section_menu: corda-enterprise-4-10
-title: Corda Enterprise Edition 4.10
+title: Corda Enterprise 4.10
 version: 'Enterprise 4.10'
 ---
 
-# Welcome to Corda Enterprise Edition 4.10
+# Corda Enterprise 4.10
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/_index.md:16]: Links on this page seem to be broken in the preview.

File: content/en/platform/corda/4.10/community/_index.md [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
-# Welcome to Corda Community Edition 4.10
+# Corda Community Edition 4.10
 
 Corda Community Edition is a free to use, open source version of Corda, with added optional support. By choosing to develop using this edition of Corda, you can access affordable technical support to help you take your project to market with confidence.
 
 {{< vimeo 205410473 >}}
 
+Corda is the world’s first private, permissioned distributed ledger technology (DLT) platform designed to work with today’s financial services industry. While regulated companies may start on public blockchains, they soon realize when they get to production, that they require capabilities native to Corda such as privacy, security, scalability, and ease-of-integration with existing systems. That’s why R3 is also exploring interoperability with assets that originate on a non-Corda network.
+
 ## What's new in Corda Community Edition
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/community/_index.md:24]: Was expecting Release Notes based on the heading.

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.10/enterprise/_index.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/_index.md:30]: Yeah let me look - I already need to get rid of that "About Corda" mention too.

Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/_index.md]: OK removed that heading and moved paragraphs around to make it read better.

File: content/en/platform/corda/4.10/community/_index.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/community/_index.md]: I removed the heading entirely as it doesn't seem to be required.

File: content/en/platform/corda/4.10/enterprise/_index.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/_index.md:16]: Fixed the links.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by AnnaK-R3 — APPROVED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**